helm/net-exporter-chart/templates/np.yaml

```diff
@@ -9,21 +9,45 @@ metadata:
 spec:
   podSelector:
     matchLabels:
-      app: {{ .Values.name }}
+      k8s-app: {{ .Values.name }}
```

Comment:
its not really k8s-app as it does not do anyhting with k8s functionality, (like kube-proxy or calico-node)

Reply:
I think you’re commenting on some outdated config